remix/.env.sample

```diff
@@ -1,7 +1,6 @@
 DATABASE_URL="postgresql://postgres:postgres@localhost:5432/mydb?schema=public"
 NODE_ENV=development
-GOOGLE_CLIENT_ID=XXXXXX
-GOOGLE_CLIENT_SECRET=XXXXXX
+GOOGLE_CLIENT_ID=274093098276-7sar8nsfcpsge9ni7p4nbi5rui6ed3dn.apps.googleusercontent.com
 GOOGLE_CLIENT_SECRET=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
```

Comment:
Same as above 🤔

Reply:
For this one I can remove it

Solved on: [5aace66](https://github.com/nimblehq/js-framework-benchmark/pull/105/commits/5aace6609fb6189d6550794b013ac455a01287c9)